iqama_fetcher.py

```diff
@@ -15,26 +17,42 @@ def GetIqamaTime(desired_prayer, masjid):
   Returns: a dict containing the prayers and iqama times
   """
 
-  # set up the parameters in the format expected by the scraper service
-  post_params = {
-    'masjid': masjid,
-  }
-  print 'scraper get_params = ', post_params
-  request = requests.post(_SCRAPER_URL, params=post_params, timeout=15)
+  masjid_encoded = util._EncodeParameter(masjid,0)
+  masjid_ID = GetIqamaID(masjid_encoded)
+  
+  desired_prayer = util.GetPrayerKeyName(desired_prayer)
+  
+  print 'Masjid ID ', masjid_ID
+
+  if not masjid_ID:
+    return None
+
+  request = requests.get(_IQAMAH_NET_URL + str(masjid_ID) + ".xml", timeout=15)
   
   # checks if request isn't bad
   if(request.status_code == requests.codes.ok):
-    response = json.loads(request.text)
-    print 'response from scraper service = ', response
-
-    # translate the dict keys to enums
-    iqama_times = {}
-    for key in response:
-      prayer = util.StringToDailyPrayer(key)
-      if prayer:
-        iqama_times[prayer] = response[key]
+    tree = ElementTree.fromstring(request.content)
+    
+    for index in range(0,len(tree[1])):
```

Comment:
instead of using indexes, what about searching by tag name, so that we're resilient to changes in the ordering of the response.